retrieval/k8s/k8s_stats.go

```diff
@@ -124,6 +126,38 @@ func writeK8sResourceFile(workDir *os.File, resourceName string,
 	return err
 }
 
+//nolint:gocyclo
+func shouldSkipResource(k8Resource interface{}) bool {
+	// safe buffer to allow for longer lived resources to be ingested correctly
+	previousHour := time.Now().Add(-1 * time.Hour)
+	switch resource := k8Resource.(type) {
+	case *v1batch.Job:
+		if resource.Status.CompletionTime != nil &&
+			previousHour.After(resource.Status.CompletionTime.Time) {
```

Comment:
Maybe a dumb question, but could the host/status file has different timezone?

Reply:
Fair concern, forced UTC

Reply:
> Fair concern, forced UTC

Is k8s timestamp always in UTC?